### PR TITLE
[#184034033]: Update documentation about URLs.

### PR DESCRIPTION
**Pull request by lublagg**

### CODE REVIEW

File: docs/launch.md [hunk 1]
```diff
@@ -46,16 +46,57 @@ In this diagram the Activity Player could be substituted for another third party
...
+https://learn.concord.org/api/v1/classes/60888
+
+Here is an example request for the JWT:
+https://learn.concord.org/api/v1/jwt/firebase?firebase_app=report-service-pro&class_hash=7883841115415ce2255ec7d0eef80f8c6e096191d63b5be1&resource_link_id=135999&target_user_id=1063089
+
+Note that the `target_user_id` parameter is passed. This is necessary so the returned JWT will include the `target_user_id` property if the researcher requesting the JWT has access to this target user's data.
+
+##### To do: Researchers, project researchers and project admins should be able to load interactive links from the details report
+
+The details report generated by report-service includes links to view the interactives with students' saved state. These links can be opened by portal wide admins or teachers in the class. The problem is that these links cannot currently be opened by portal wide admins, project researchers, or project admins.
```
Comment (scytacki) [inline docs/launch.md:65]: I think this has a typo in it. I think portal wide admins can open these links.

**Review by scytacki — CHANGES_REQUESTED** [review]
I think there is an error (probably written by me) in the text. See the specific comment.
Can you check wether portal wide admins can or cannot open these links?

File: docs/launch.md (hunk 1 above)
Comment (lublagg) [reply docs/launch.md]: Gotcha. Portal-wide admins can open the links. I think it's just project researchers and project admins (although I'm not sure what a project admin is, actually).

**Review by scytacki — APPROVED** [review]
Looks good to me.

**Pull request merged**